ext/lib/mgmt/mcumgr/transport/smp-bluetooth.md

```diff
@@ -0,0 +1,50 @@
+# SMP over Bluetooth
```

Comment:
@carlescufi what do we do about all this nice documentation? It is not integrated as part of zephyr docs. (fyi @dbkinder )

Reply:
@nashif I'm looking into including .md files such as these in the /ext directory, within the Sphinx doc tree. But it looks like only commonmark is support (so things like tables aren't rendered correctly).  Sigh.  This SMP over Bluetooth .md file has a table.